package.json

```diff
@@ -6,7 +6,9 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "tsc -p tsconfig.json",
+    "clean": "tsc --build tsconfig.build.json --clean",
```

Comment:
i forgot to add cleaning step in #88

Reply:
aren't these backwards? build should just build, and rebuild should be clean + build?

Reply:
no, `tsc --build tsconfig.build.json --clean` doesn't actually emit any files, it just looks at out dir from set `tsconfig` and clears it (it works for single and multi projects)

in case of this repo there is no difference between `tsc -p tsconfig.json` and `tsc --build tsconfig.json` as there is only one project

Reply:
Right... I mean the names for the scripts "rebuild" and "build" are swapped.

Reply:
right, i didn't think about that, I'm bad at naming stuff, fixed :)